src/transformer/utils/get_user.php

```diff
@@ -19,8 +19,9 @@
 
 function get_user(array $config, \stdClass $user) {
     $fullname = get_full_name($user);
+    $hasValidEmail = filter_var($user->email, FILTER_VALIDATE_EMAIL);
```

Comment:
@AndyHubert can you please rename `hasValidEmail` to `hasvalidemail` in order to pass Moodle's linting rules. Very unfortunate that they don't allow camel-cased variables.